psi4/driver/procrouting/proc_table.py

```diff
@@ -206,15 +206,19 @@
 
 # Integrate DFT with driver routines
 for ssuper in proc.dft_funcs.superfunctional_list:
-      procedures['energy'][ssuper.name().lower()] = proc.run_dft
+      procedures['energy'][ssuper.name().lower()] = proc.run_scf
```

Comment:
I definitely agree with HF/DFT path unification, but could you remind us what the new core determiner of HF vs. DFT will now be since not this? RHF/UHF/ROHF vs. RKS/UKS?

Reply:
Basically it comes down to if the kwarg is passed or not.